widgetry/src/backend_glow.rs

```diff
@@ -418,180 +476,170 @@ impl PrerenderInnards {
 ///
 /// Implementation is based on the the description of ArrayTextures from:
 /// https://www.khronos.org/opengl/wiki/Array_Texture.
-pub fn load_textures(
```

Comment:
This method transposes bytes from a spritesheet and then uploads them to the graphics card. It made some assumptions about OpenGL versions which weren't compatible with WebGL 1.0. 

Even though I didn't end up finishing it, I'd like to keep this change which separates the "format the bytes" from the "upload to open GL" functionality. I think it's clearer and would be helpful if we eventually do implement textures for WebGL 1.0